File: 3648/3648.py
class Solution:
    def minimum_sensors_to_cover_grid(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.minimum_sensors_to_cover_grid(root.left), self.minimum_sensors_to_cover_grid(root.right))

File: 3648/3648.go
func minimumSensorsToCoverGrid(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumSensorsToCoverGrid(root.Left)
	r := minimumSensorsToCoverGrid(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}